static/src/js/components/CollectionResults/CollectionResultsHeader.js

```diff
@@ -161,17 +162,20 @@ const CollectionResultsHeader = ({
             </Col>
           </Form.Group>
         </Row>
-        <div className="row mt-1">
-          <div className="col">
-            <span className="collection-results-header__tip">
-              <strong className="collection-results-header__tip-label">Tip:</strong>
-              Add
-              <i className="collection-results-header__tip-icon fa fa-plus" />
-              collections to your project to compare and download their data.
-            </span>
-          </div>
-        </div>
-
+        {
+          authenticationEnabled(portal) && (
```

Comment:
Can we move this to a component that looks at the portal value from the store and wraps the conditional components, passing them as children? The component can be connected.